<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro de pacientes</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Dados do paciente</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="1">
           <widget class="QLineEdit" name="cpf">
            <property name="inputMask">
             <string>###.###.###-##</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>RG</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="rg">
            <property name="inputMask">
             <string>##########</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QComboBox" name="combo_plano">
            <item>
             <property name="text">
              <string>Sim</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Não</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QPushButton" name="excluir_btn">
            <property name="text">
             <string>Excluir</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Nome</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Plano de saúde</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QPushButton" name="salvar_btn">
            <property name="text">
             <string>Salvar</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Idade</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLineEdit" name="nome"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Sexo</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="fone">
            <property name="inputMask">
             <string>(##)#####-####</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QComboBox" name="combo_sexo">
            <item>
             <property name="text">
              <string>Masculino</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Feminino</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLineEdit" name="idade">
            <property name="inputMask">
             <string>##</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Telefone</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>CPF</string>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QPushButton" name="limpar_btn">
            <property name="text">
             <string>Limpar dados</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>../icons/foto cadastro.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="margin">
           <number>-50</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Pacientes cadastrados</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nome</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Idade</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Telefone</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>CPF</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>RG</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Sexo</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Plano</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nome</tabstop>
  <tabstop>cpf</tabstop>
  <tabstop>rg</tabstop>
  <tabstop>idade</tabstop>
  <tabstop>fone</tabstop>
  <tabstop>combo_sexo</tabstop>
  <tabstop>combo_plano</tabstop>
  <tabstop>salvar_btn</tabstop>
  <tabstop>excluir_btn</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
